limitlessled/group/commands/legacy.py

```diff
@@ -0,0 +1,190 @@
+    BRIGHTNESS_STEPS = 10
+    TEMPERATURE_STEPS = 10
+
+    def __init__(self, bridge, group_number):
+        """
+        Initializes the command set.
+        :param bridge: The bridge the leds are connected to.
+        :param group_number: The group number.
+        """
+        super().__init__(bridge, group_number, self.BRIGHTNESS_STEPS,
+                         temperature_steps=self.TEMPERATURE_STEPS)
+
+    def on(self):
+        """
+        Build command for turning the led on.
+        :return: The command.
+        """
+        return self._build_command(self.ON_BYTES[self._group_number - 1])
+
+    def off(self):
+        """
+        Build command for turning the led off.
+        :return: The command.
+        """
+        return self._build_command(self.OFF_BYTES[self._group_number - 1])
+
+    def dimmer(self):
+        """
+        Build command for setting the brightness one step dimmer.
+        :return: The command.
+        """
+        return self._build_command(0x34, select=True, select_command=self.on())
+
+    def brighter(self):
+        """
+        Build command for setting the brightness one step brighter.
+        :return: The command.
+        """
+        return self._build_command(0x3C, select=True, select_command=self.on())
+
+    def cooler(self):
+        """
+        Build command for setting the temperature one step cooler.
+        :return: The command.
+        """
+        return self._build_command(0x3F, select=True, select_command=self.on())
+
+    def warmer(self):
+        """
+        Build command for setting the temperature one step warmer.
+        :return: The command.
+        """
+        return self._build_command(0x3E, select=True, select_command=self.on())
+
+
+class CommandSetRgbwLegacy(CommandSetLegacy):
+    """ Command set for RGBW led light connected to legacy wifi bridge. """
+
+    SUPPORTED_LED_TYPES = [RGBW]
+    HUE_STEPS = 255
+    BRIGHTNESS_STEPS = 25
+
+    def __init__(self, bridge, group_number):
+        """
+        Initializes the command set.
+        :param bridge: The bridge the leds are connected to.
+        :param group_number: The group number.
+        """
+        super().__init__(bridge, group_number, self.BRIGHTNESS_STEPS,
+                         hue_steps=self.HUE_STEPS)
+
+    def on(self):
+        """
+        Build command for turning the led on.
+        :return: The command.
+        """
+        return self._build_command(self._offset(0x45))
+
+    def off(self):
+        """
+        Build command for turning the led off.
+        :return: The command.
+        """
+        return self._build_command(self._offset(0x46))
+
+    def white(self):
+        """
+        Build command for turning the led into white mode.
+        :return: The command.
+        """
+        return self._build_command(self._offset(0xC5),
+                                   select=True, select_command=self.on())
+
+    def color(self, color):
```

Comment:
This class is missing the `hue` command as required by `limitlessled/group/rgbw.py`. This might be an easy fix for you since you're most familiar with the `hue` stuff.

I will also take a look on my side.